### ISSUE TITLE
"Unknown word" error for interpolated value with function

### ISSUE THREAD

**Issue by niksy** [stylelint/stylelint#1640]
> Bug description

When trying to use function inside SCSS interpolation, stylelint reports syntax error with message "Unknown word".

```sh
index.scss
 17:44  ✖  Unknown word   CssSyntaxError
```

> How to reproduce

Input SCSS:

```scss
.foo {
	$var:foo;
	background-image:#{$var}bar;
}

.bar {
	$var:to-upper-case(foo);
	background-image:#{$var}bar;
}

.bad {
	$var:to-upper-case(foo);
	background-image:url(#{$var}bar);
}

.baz {
	background-image:url(#{to-upper-case(foo)}bar); // This is where error is triggered
}
```

Output CSS with node-sass 3.8.0:

```css
.foo {
  background-image: foobar; }

.bar {
  background-image: FOObar; }

.bad {
  background-image: url(FOObar); }

.baz {
  background-image: url(FOObar); }
```

---

Using latest stylelint with SCSS parser.

Is this related to stylelint or postcss-scss?

**Comment by davidtheclark** [comment]
It's almost certainly related to postcss-scss, because we're getting a syntax error. stylelint doesn't do any parsing itself, so never throws syntax errors: they always come from upstream.

**Comment by niksy** [comment]
OK, I’ve created the issue on postcss-scss repo so we will se how it goes.

**Comment by evilebottnawi** [comment]
Ref: https://github.com/postcss/postcss-scss/issues/36

**Comment by ai** [comment]
Fixed https://github.com/postcss/postcss-scss/issues/38

Release will be on this week (I need to fix one other interpolation issue too).

**Comment by evilebottnawi** [comment]
Close. Fixed https://github.com/postcss/postcss-scss/commit/01bb9da154cd3ee97932774d2ea3c26099b50a72

**Comment by ai** [comment]
Released in PostCSS SCSS 0.1.9

**Comment by ntwb** [comment]
_Make sure to run `npm update` to update `postcss-scss` to get this fix (it's covered by stylelints semantic versioning)_
